src/tests/ftest/util/apricot/apricot/__init__.py

```diff
@@ -0,0 +1,5 @@
+''' apricot __init__ '''
+__all__ = ['Test', 'TestWithServers', 'skipForTicket']
+
+from apricot.test import Test, TestWithServers, TestWithoutServers
+from apricot.test import TestWithServersAndClients, skipForTicket
```

Comment:
(pylint-no-name-in-module) No name 'TestWithServersAndClients' in module 'apricot.test'